docs/data/components/field/UnstyledFieldServerError.js

```diff
@@ -39,11 +39,10 @@ export default function UnstyledFieldServerError() {
 
   return (
     <form onSubmit={handleSubmit} noValidate>
-      <FieldRoot invalid={error}>
+      <FieldRoot invalid={error} name="email">
```

Comment:
`name` and `diasbled` were moved back onto `<Field.Root>` for SSR/layout shift reasons with the server-side `errors` prop and style hooks.

We should always try to avoid using effects to sync up data for this reason, unless it's a popup element (which requires interaction to appear for the most part, or at least until hydration which won't cause layout shift).